src/openvino/interface.cpp

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Is there anything OpenVINO device specific? Or is it all just CPU?

Reply:
Right now it is all just CPU. Eventually we may add something more OpenVINO-specific.

Reply:
Is it from a separate allocator or is it the CPU allocator?

Reply:
Could you elaborate?

Reply:
Is the memory used by OpenVINO from the default CPU allocator? Or does it have a separate OpenVINO OrtAllocator?

Reply:
Ah, yes we are using the default CPU allocator.

Reply:
Ah, then there shouldn't be an OpenVINOMemory object yet, as this would only be if you had your own OrtAllocator. CPU memory is already managed by the CpuMemory object.

Reply:
So then I guess you're saying that our Generators::OpenVINO::DeviceInterface should implement AllocateBase / WrapMemoryBase by creating CpuMemory objects?

Reply:
okay, this should be resolved in latest push.